qa/integration/services/dockerized/Dockerfile

```diff
@@ -16,6 +16,11 @@ USER tester
 ENV WORKDIR /home/tester
 WORKDIR $WORKDIR
 
+# Generate TLS certificates that can be used by services.
+ENV CERTIFICATES_DIR $WORKDIR/certificates
+RUN mkdir -p $CERTIFICATES_DIR && \
```

Comment:
I don't have the rationale but certificates used to committed in the repo. The PR#5928 deleted the files so that now filebeat test generate them on-the-fly. Perhaps @suyograo knows the reason?

About the other doubt, I don't know if the scenario of build two child images from different versions of the base can even happen.
One intermediate solution (neither generate in the image nor commit in repo) can be a helper that create the certificates in a temporary folder in host. Then it's up to tests requiring them to copy over the containers.